src/containerapp/azext_containerapp/_clients.py

```diff
@@ -15,6 +15,8 @@
 logger = get_logger(__name__)
 
 STABLE_API_VERSION = "2022-03-01"
```

Comment:
Nit: if stable api-version isn't being used any where can we remove this constant?